.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,63 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Build release distribution"
+        run: make release
+        env:
+          GITHUB_ACTOR: ${{ github.actor }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "Extract version information from project"
+        run: |
+          MARKETING_VERSION=$(xcrun xcodebuild -showBuildSettings -project Sparkle.xcodeproj | grep MARKETING_VERSION | cut -f 7 -d ' ')
+          echo ::set-env name=MARKETING_VERSION::${MARKETING_VERSION}
+
+      - name: "Determine if this is a pre-release version"
+        run: |
+          SPARKLE_VERSION_SUFFIX=$(xcrun xcodebuild -showBuildSettings -project Sparkle.xcodeproj | grep SPARKLE_VERSION_SUFFIX | cut -f 7 -d ' ')
+          if [[ -z "$SPARKLE_VERSION_SUFFIX" ]]; then
+            echo ::set-env name=PRERELEASE_VERSION::true
+          else
+            echo ::set-env name=PRERELEASE_VERSION::false
+          fi
+
+      - name: "Commit and push updated Swift package description"
+        run: |
+            git add Package.swift
+            git commit -m "Update Package.swift for version ${{ env.MARKETING_VERSION }}"
```

Comment:
The push is necessary but the add and commit are not necessary if release-move-tag.sh does this. In the future I may want to add/commit the podspec file as well, and release-move-tag.sh seems to be a centralized place to make such a change.

Reply:
I've updated this behaviour in 3e943bf5f7b45286ab50353032561748f8c33079, but is there still a need to move the tag at all?

Reply:
See https://github.com/sparkle-project/Sparkle/pull/1908/files#r687424758